tmt/steps/provision/artemis.py

```diff
@@ -455,6 +455,13 @@ def api(self) -> ArtemisAPI:
 
         return self._api
 
+    @property
+    def is_ready(self) -> bool:
+        """ Detect the guest is ready or not """
+
+        # XXX: How to identify GuestArtemis is ready ???
```

Comment:
The following should be good enough:

```python
return self.guest is not None
```

It's trivial but not perfect, please, add a TODO comment for me here, I'll submit a more robust solution later - Artemis is still more of a dogfooding project than being widely used, so no need for you to dive into its internals. Small refactoring of the plugin will be necessary.

Reply:
Hi @happz, thank you for your suggestion! I have updated the related `is_ready`  in `tmt/steps/provision/artemis.py`, please help to take a look again, thanks!